packages/lib-runtime/src/store/usePluginInfo.ts

```diff
@@ -0,0 +1,20 @@
+import * as _ from 'lodash-es';
+import type { PluginInfoEntry } from '../types/store';
+import { PluginEventType } from '../types/store';
+import { usePluginSubscription } from './usePluginSubscription';
+
+/**
+ * TODO docs
+ */
+export const usePluginInfo = (): PluginInfoEntry[] => {
+  const infoEntries = usePluginSubscription(
+    (pluginConsumer) => pluginConsumer.getPluginInfo(),
+    _.isEqual,
+    [PluginEventType.PluginInfoChanged],
+  );
+
+  return infoEntries;
+};
+
+export const noPluginsPending = (infoEntries: PluginInfoEntry[]) =>
```

Comment:
Interesting, do you see this being something they want to do outside of `usePluginInfo`? I'm thinking we can just make this a hook that internally looks it up through `usePluginInfo`.

Reply:
FYI, I've removed the helper `noPluginsPending` function in the latest commit, since code like

```ts
infoEntries.every((e) => e.status !== 'pending')
```

isn't really infra or SDK related, it's just processing of the `usePluginInfo` hook result, which can be done directly in the consuming code.

Reply:
> Interesting, do you see this being something they want to do outside of `usePluginInfo`?

It's code that post-processes the result of `usePluginInfo` hook.

If we think it has high value to consumers, we can make a hook out of it. Personally, I'd leave this to the consumers.